### DISCUSSION TITLE
Not a problem just a question

### DISCUSSION THREAD

**Discussion by Pseudow** [compnerd/swift-win32#738] [General]
Hi, I have recently started using Swift on Windows but the fact is that there is absolutely no library for things like http and windows sockets and I was wondering how did you make to import WinSDK into Swift. I’m currently looking into the code but I can’t find it if you could help me a little bit I am not really familiar with CMake that could explain it but just if you could redirect me to a file it would be really helpful.

**Comment by compnerd** [comment]
This is better suited to the Swift Forums as this doesn't really pertain to swift-win32.  However, the answer is that WinSDK is part of the SDK and is a clang module, not part of swift-win32.
